### DISCUSSION TITLE
Support to decode McCain ATC eX .dat logs

### DISCUSSION THREAD

**Discussion by oschwartz10612** [udotdevelopment/ATSPM#220] [Q&A] [answered]
Hi,

Is it possible to decode these logs, or do I need to reach out to the vendor to get their decoder program to reverse engineer?

Best,
Owen

**Comment by avetraffic** [comment, marked as answer]
Hi, oschwartz,

Right now ATSPM doesn't support McCain ATC right out of the box. So like you mentioned you would need to reach out to McCain to get access to their decoder.

**Comment by oschwartz10612** [comment]
Hi,

Thanks for the response. We are in the process of doing just this. 

Best, 
Owen
